corehq/warehouse/migrations/0004_app_status_fact_form_and_sync_staging_and_deleted_field.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-25 09:26
+from __future__ import unicode_literals
+
+import corehq.warehouse.models.shared
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('warehouse', '0003_userstagingtable'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ApplicationStatusFact',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('last_form_submission_date', models.DateTimeField(null=True)),
+                ('last_sync_log_date', models.DateTimeField(null=True)),
+                ('last_form_app_build_version', models.CharField(max_length=255)),
+                ('last_form_app_commcare_version', models.CharField(max_length=255)),
+                ('last_form_app_source', models.CharField(max_length=255)),
+                ('last_sync_log_app_build_version', models.CharField(max_length=255)),
+                ('last_sync_log_app_commcare_version', models.CharField(max_length=255)),
+                ('last_sync_log_app_source', models.CharField(max_length=255)),
+            ],
+            bases=(models.Model, corehq.warehouse.models.shared.WarehouseTableMixin),
+        ),
+        migrations.CreateModel(
+            name='FormStagingTable',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('form_id', models.CharField(max_length=255, unique=True)),
+                ('domain', models.CharField(default=None, max_length=255)),
+                ('app_id', models.CharField(max_length=255, null=True)),
+                ('xmlns', models.CharField(default=None, max_length=255)),
+                ('user_id', models.CharField(max_length=255, null=True)),
+                ('received_on', models.DateTimeField(db_index=True)),
+                ('build_id', models.CharField(max_length=255, null=True)),
```

Comment:
this actually might be a bit unrealistic right now... it means we'd have to fetch the form from riak to get the time_start and time_end since we don't save it to the form